test/topology_experimental_raft/test_topology_ops.py

```diff
@@ -62,14 +63,11 @@ async def test_topology_ops(request, manager: ManagerClient):
     await wait_for_cql_and_get_hosts(cql, servers, time.time() + 60)
     gen_timestamps = {r.time for r in await manager.get_cql().run_async(query)}
     logger.info(f"Timestamps before check_and_repair: {gen_timestamps}")
-    await manager.api.client.post("/storage_service/cdc_streams_check_and_repair", servers[1].ip_addr)
-    async def new_gen_appeared() -> Optional[set[datetime]]:
-        new_gen_timestamps = {r.time for r in await manager.get_cql().run_async(query)}
-        assert(gen_timestamps <= new_gen_timestamps)
-        if gen_timestamps < new_gen_timestamps:
-            return new_gen_timestamps
-        return None
-    new_gen_timestamps = await wait_for(new_gen_appeared, time.time() + 60)
+    await asyncio.gather(*[manager.api.client.post("/storage_service/cdc_streams_check_and_repair", servers[i % 2].ip_addr)
+                          for i in range(10)])
+    time.sleep(60)
+    new_gen_timestamps = {r.time for r in await manager.get_cql().run_async(query)}
+    assert(len(gen_timestamps) + 1 == len(new_gen_timestamps))
```

Comment:
I propose you use the `wait_for` function (test/pylib/util.py):
- wait until at least one new generation appears (with a timeout of 60 seconds, for example)
- then assert that only one new generation has appeared.

Reply:
I did that. The problem with this approach is that we might potentially omit a second generation creation that happened later. However, since sleep (or anything equivalent) is unacceptable, I don't think we can do a better test here.

Reply:
> I did that. The problem with this approach is that we might potentially not detect a second generation creation that happened later. 

Yep, although the nice thing is, with multiple runs at least some of them will detect this second generation when it happens, and the detection should have high probability. So I think it's fine.